test/Groups/forms.jl

```diff
@@ -25,7 +25,7 @@
    @test gram_matrix(f)==B
    @test ishermitian_form(f)
    @test f.mat_iso isa MapFromFunc
-   @test f.X isa GapObj
+   @test f.X isa GAP.GapObj
```

Comment:
Then perhaps also remove `using GAP` from `test/runtest.jl`?

Reply:
I do not know what a good policy would be. Calling `using` in `test/runtests.jl` means that these statements are not executed if one runs just a subset of the tests. Removing such central `using` statements means that all testfiles will be affected. Is it perhaps more "Julia like" to put `using` statements only into the individual testfiles?

Reply:
No, that's IMHO not a good idea, as the using statement will affect all files included later on, and that can lead to accidental hidden dependencies in those other test files.

So I'd argue that `using` should only be used in runtest.jl or not at all. And here I'd rather not have the `using GAP` at all, as it serves no purpose (other than allowing us to omit the `GAP.` prefix, which you are eliminating here.

Reply:
Fine. Then I will remove `using GAP` from `test/runtest.jl`.